src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/Example1.java

```diff
@@ -0,0 +1,58 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="Indentation"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;
+
+// xdoc section -- start
+class Example1 {
+    String field = "example";                                      // basicOffset
+    int[] values = {                                                // basicOffset
+        10,
+        20,
+        30
+    };
+
+    void processValues() throws Exception {
+        handleValue("Test String", 42);                              // basicOffset
+    }
+
+    void handleValue(String aFooString,
+                     int aFooInt) {                                 // indent:8 ; expected: > 4; ok, because 8 > 4
+        boolean cond1 = true;
+        boolean cond2 = false;
+        boolean cond3 = true;
+        boolean cond4 = false;
+        boolean cond5 = true;
+        boolean cond6 = false;
+
+        if (cond1
+            || cond2) {
+            field = field.toUpperCase()
+                .concat(" TASK");
+        }
+
+        if ((cond1 && cond2)
+                || (cond3 && cond4)                              // ok, lineWrappingIndentation
+                || !(cond5 && cond6)) {                           // ok, lineWrappingIndentation
+            field.toUpperCase()
+                 .concat(" TASK")                                 // ok, lineWrappingIndentation
+                 .chars().forEach(c -> {                          // ok, lineWrappingIndentation
+                     System.out.println((char) c);
+                 });
```

Comment:
```
        if ((cond1 && cond2)
                || (cond3 && cond4)                // ok, lineWrappingIndentation
                || !(cond5 && cond6)) {            // ok, lineWrappingIndentation
            field.toUpperCase()
                 .concat(" TASK")                  // ok, lineWrappingIndentation
                 .chars().forEach(c -> {           // ok, lineWrappingIndentation
                     System.out.println((char) c);
```

please move comments blocks close to code, by blocks

Reply:
done

Reply:
Alignment of comments please

Reply:
done